Jenkinsfile

```diff
@@ -167,7 +167,7 @@ pipeline {
 
                     environment {
                         TESTBED = credentials('win-testbed')
-                        TESTBED_TEMPLATE = "Template-testbed-201902010453"
+                        TESTBED_TEMPLATE = "Template-testbed-201903181426"
```

Comment:
What was changed in the testbed?

Reply:
Newest Windows updates and python:3.7 docker image.